src/Umbraco.Cms.Api.Common/OpenApi/SwaggerRouteTemplatePipelineFilter.cs

```diff
@@ -63,7 +63,6 @@ protected virtual string SwaggerUiRoutePrefix(IApplicationBuilder applicationBui
         }
 
         // Add custom configuration from https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/
-        swaggerUiOptions.ConfigObject.PersistAuthorization = true; // persists authorization data so it would not be lost on browser close/refresh
```

Comment:
I would personally perfer this to stay. When you reboot many times it is very handy not to have to sign in every time.
Not sure if we can figure out a way to set a timeout there too

Reply:
There seems to be no option for a timeout here. But let's just keep it for now. It will cause annoyance if it's gone, and annoyance if it's there. Catch22 😄 if people get really annoyed with the auth persistence, we can always remove it.